text-ui-test/runtest.bat

```diff
@@ -0,0 +1,21 @@
+@ECHO OFF
```

Comment:
I like how you try to automate the testing by having an input file, actual and expected output. *thumbs up*